terraform/table_terraform_module.go

```diff
@@ -0,0 +1,135 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/Checkmarx/kics/pkg/model"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+func tableTerraformModule(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "terraform_module",
+		Description: "Terraform module information.",
+		List: &plugin.ListConfig{
+			ParentHydrate: tfConfigList,
+			Hydrate:       listModules,
+			KeyColumns:    plugin.OptionalColumns([]string{"path"}),
+		},
+		Columns: []*plugin.Column{
```

Comment:
Can you please add the `end_line` and `source` columns like we have in other tables? Beside adding the column definitions, I believe this should be the main code you need to add to the `buildModule` function - https://github.com/turbot/steampipe-plugin-terraform/blob/main/terraform/table_terraform_data_source.go#L167-L175.

Reply:
Again: thanks & makes sense, also thanks for directly pointing me to a good example to (re-)use. Fixed with latest commit.